src/plugins/controls/common/range_slider/types.ts

```diff
@@ -13,7 +13,7 @@ export const RANGE_SLIDER_CONTROL = 'rangeSliderControl';
 export type RangeValue = [string, string];
 
 export interface RangeSliderEmbeddableInput extends DataControlInput {
-  value: RangeValue;
+  value?: RangeValue;
```

Comment:
This fixes an inconsistency where, even though this was marked as a required part of `RangeSliderEmbeddableInput`, it could actually still sometimes be undefined - this makes it easier to catch bugs where we aren't being type safe.